src/App.tsx

```diff
@@ -3,19 +3,29 @@ import { useSelector } from 'react-redux';
 
 import './App.scss';
 import { Start } from './components/Start';
+import { PostsList } from './components/PostsList';
+import { SearchField } from './components/SearchField';
 
-import { isLoading, getMessage } from './store';
+import { isLoading, getPosts } from './store';
 
 const App = () => {
   const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+  const posts = useSelector(getPosts);
 
   return (
     <div className="App">
-      <h1>Redux list of posts</h1>
-      <h2>{loading ? 'Loading...' : message}</h2>
 
-      <Start />
+      {
+        !posts.length
+          ? <Start />
+          : (
+            <>
+              <SearchField />
+              <PostsList />
+            </>
+          )
+      }
+      <h2>{loading && 'Loading...'}</h2>
```

Comment:
```suggestion
      {loading && <h2>{ 'Loading...' }</h2>}
```
![image](https://user-images.githubusercontent.com/60401137/88643671-ac3ef600-d0ca-11ea-80f0-5ce960ea5b41.png)